src/pyhf/infer/intervals.py

```diff
@@ -54,9 +54,10 @@ def upperlimit(data, model, scan, level=0.05, return_results=False):
               :class:`~pyhf.infer.hypotest` results at each test POI.
               Only returned when ``return_results`` is ``True``.
     """
+    test_stats = kwargs.pop('test_stat', "qtilde")
```

Comment:
should we maybe rename `kwargs` into `hypotest_kwargs` and pass

```
hypotest(mu, data, model, return_expected_set=True, **hypotest_kwargs)
```

Reply:
The idea being here that this allows for further manipulation of `hypotest` from `upperlimit` as desired? Seems to make sense.

Reply:
Yeah

Reply:
@lukasheinrich @matthewfeickert Alright I'll pass the ```hypotest_kwargs``` to infer/hypotest directly.